pkg/sql/execinfrapb/data.proto

```diff
@@ -378,3 +379,20 @@ message DistSQLVersionGossipInfo {
 message DistSQLDrainingInfo {
   optional bool draining = 1 [(gogoproto.nullable) = false];
 }
+
+// TracingAggregatorEvents represent a collection of events emitted by a tracing
+// aggregator. These events contain the execution details of the component being
+// monitored by the tracing aggregator.
+message TracingAggregatorEvents {
+  // NodeID is the node from which the aggregator event is being pushed to the
+  // coordinator of the job.
+  optional int32 node_id = 1 [(gogoproto.nullable) = false,
+    (gogoproto.customname) = "NodeID",
```

Comment:
I suppose this could be SQLInstanceID but I don't feel strongly.

Reply:
good point, done